functions.php

```diff
@@ -124,12 +124,22 @@ function bootscore_widgets_init() {
       'name'          => esc_html__('Top Nav', 'bootscore'),
       'id'            => 'top-nav',
       'description'   => esc_html__('Add widgets here.', 'bootscore'),
-      'before_widget' => '<div class="ms-3">',
+      'before_widget' => '<div class="top-nav-widget ms-2">',
       'after_widget'  => '</div>',
       'before_title'  => '<div class="widget-title d-none">',
       'after_title'   => '</div>'
     ));
-    // Top Nav End
+    
+    // Top Nav 2
+    register_sidebar(array(
+      'name'          => esc_html__('Top Nav 2', 'bootscore'),
+      'id'            => 'top-nav-2',
+      'description'   => esc_html__('Add widgets here.', 'bootscore'),
```

Comment:
Maybe give a better description about where and when these widgets are being loaded?

Reply:
Why? It's just a widget?

Reply:
Yeah it's a widget area, but this one will go in both navbar and offcanvas, while the other stays in navbar. That would be things that may be nice to set in the description, as you see those while adding widgets, correct? Makes it easier to find where to put stuff, also for devs. Get lost myself often enough. :P

Reply:
Done